build/templates/_converters.py.mako

```diff
@@ -222,6 +222,36 @@ def convert_init_with_options_dictionary(values, encoding):
     return init_with_options_string
 
 
+% if config['module_name'] == 'nitclk':
+# nitclk specific converters
+def convert_to_nitclk_session_num(item):
+    '''Convert from supported objects to NI-TClk Session Num
+
+    Supported objects are:
+    - class with .tclk object of type nitclk.SessionReference
+    - nitclk.SessionReference
+    - NI-TClk Session Num
+    '''
+    try:
+        return item.tclk.get_session_number()
+    except KeyError:
+        pass
+
+    try:
+        return item.get_session_number()
+    except KeyError:
+        pass
+
+    # If we haven't gotten a SessionReference, we assume the item is the actual nitclk session num and return it
+    return item
+
+
+def convert_to_nitclk_session_num_list(item_list):
```

Comment:
`convert_session_reference_list_to_vi_session_list` ?

Reply:
The type is `ViSession`, but more specifically it is the nitclk session number. For some APIs this is different than the ViSession that is the main session number. This is why I named it that. If that is a special enough case, I can change it, but it may be misleading in some instances.

Reply:
ok... so "session num" is to mean "ViSession used by NI-TClk to refer to an instrument session.

Well, at least spell out `number`.

Reply:
Changed to `number`